### ISSUE TITLE
Handle missing set children more gracefully

### ISSUE THREAD

**Issue by moewew** [plk/biber#241]
See https://tex.stackexchange.com/q/458426/35864

Consider the following MWE
```
\documentclass{article}
\usepackage{biblatex}
\addbibresource{biblatex-examples.bib}

\begin{document}
\begin{refsection}
  \defbibentryset{test}{nonexistent-key}
  \printbibliography[heading=none]
\end{refsection}
\textcite{sigfridsson}
\end{document}
```

With an obviously undefined `test` entry child.

Biber produces the output
```
INFO - This is Biber 2.12 (beta)
INFO - Logfile is 'kskjhkshjshsksjs.blg'
INFO - Reading 'kskjhkshjshsksjs.bcf'
INFO - Found 0 citekeys in bib section 0
INFO - Found 1 citekeys in bib section 1
INFO - Found 1 citekeys in bib section 0
INFO - Processing section 0
INFO - Looking for bibtex format file 'biblatex-examples.bib' for section 0
INFO - LaTeX decoding ...
INFO - Found BibTeX data source 'C:/Program Files/MiKTeX 2.9/bibtex/bib/biblatex/biblatex-examples.bib'
INFO - Overriding locale 'en-US' defaults 'normalization = NFD' with 'normalization = prenormalized'
INFO - Overriding locale 'en-US' defaults 'variable = shifted' with 'variable = non-ignorable'
INFO - Sorting list 'nty/global//global/global' of type 'entry' with template 'nty' and locale 'en-US'
INFO - No sort tailoring available for locale 'en-US'
INFO - Processing section 1
Use of uninitialized value $alias in hash element at C:\Users\Moritz\AppData\Local\Temp\par-4d6f7269747a\cache-b48fe7ad8eda7c6732ebfb7751532fa7c1d79c99\inc\lib/Biber/Section.pm line 525.
Use of uninitialized value $citekey in hash element at C:\Users\Moritz\AppData\Local\Temp\par-4d6f7269747a\cache-b48fe7ad8eda7c6732ebfb7751532fa7c1d79c99\inc\lib/Biber/Entries.pm line 57.
Can't call method "datafields" on an undefined value at C:\Users\Moritz\AppData\Local\Temp\par-4d6f7269747a\cache-b48fe7ad8eda7c6732ebfb7751532fa7c1d79c99\inc\lib/Biber/Entry.pm line 534.
WARN - I didn't find a database entry for dynamic set member 'nonexistent-key' - ignoring (section 1)
```
and aborts compilation. The relevant messages don't appear in the `.blg` file, though. The `.bbl` is completely empty.

Would it be possible to have Biber handle undefined set children slightly more gracefully? After all one missing entry usually does not stop processing of the entire file.

**Comment by plk** [comment]
This was a bug when a dynamic set had non-existent keys removed so that it was completely empty after datasource fetching (adding a good key to the set in the example has no error). Fixed in 2.13 DEV.

**Comment by moewew** [comment]
Much better with 2.13. Thank you very much.
